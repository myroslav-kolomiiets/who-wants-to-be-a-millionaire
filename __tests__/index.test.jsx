import '@testing-library/jest-dom'
import { render, screen } from '@testing-library/react'
import Home from '../src/pages/index';

describe('Home', () => {
  it('renders a heading', () => {
    render(<Home />)

    const heading = screen.getByRole('heading', { level: 1 })

    expect(heading).toBeInTheDocument()
  })
})

it('renders homepage unchanged', () => {
  const { container } = render(<Home />)
  expect(container).toMatchSnapshot()
})
