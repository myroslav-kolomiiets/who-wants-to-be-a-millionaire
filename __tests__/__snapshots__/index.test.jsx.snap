// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <main
    class="homepage"
  >
    <div
      class="homepage__image"
    >
      <img
        alt="logo"
        data-nimg="1"
        decoding="async"
        fetchpriority="high"
        height="367"
        src="/images/logo.svg"
        style="color: transparent; object-fit: contain;"
        width="624"
      />
    </div>
    <div
      class="homepage__link"
    >
      <div
        class="homepage__link-wrapper"
      >
        <h1
          class="homepage__link-h1"
        >
          Who wants to be
          <br />
          a millionaire?
        </h1>
        <div
          class="homepage__link-button"
        >
          <a
            class="control control--size--responsive control--theme--primary"
            href="/game"
          >
            Start
          </a>
        </div>
      </div>
    </div>
  </main>
</div>
`;
